packages/angular-material/src/controls/number.renderer.ts

```diff
@@ -135,6 +135,7 @@ export class NumberControlRenderer extends JsonFormsControl {
       if (this.locale === undefined || this.locale !== currentLocale) {
         this.locale = currentLocale;
         this.numberFormat = new Intl.NumberFormat(this.locale, {
+          useGrouping: this.uischema.options ? this.uischema.options.useGrouping : undefined,
```

Comment:
This is fine in general but should be checked with the `config` object as it serves as a default.

Reply:
See for example the [MaterialListWithDetailRenderer](https://github.com/eclipsesource/jsonforms/blob/cd457e1af2a7515263661c833fd2e833111a194a/packages/material/src/additional/MaterialListWithDetailRenderer.tsx#L98)

Reply:
Thanks. I see your requested changes have already been implemented on #1672